addons/point_of_sale/models/uom.py

```diff
@@ -1,31 +1,34 @@
 from odoo import api, fields, models
 
 
-class UomCategory(models.Model):
-    _name = 'uom.category'
-    _inherit = ['uom.category', 'pos.load.mixin']
+class UomUom(models.Model):
+    _name = 'uom.uom'
```

Comment:
```suggestion
```
Shouldn't `_name` be set only for the base model?

Reply:
It looks like the `[REV] *: Restore the model _name` revert https://github.com/odoo/odoo/pull/186906/commits/1bf66d2f81c414e5943455cd39d61e41e579fd80 (another big change)  added in a bunch of these where they didn't exist before. In theory the `_name` can be overridden to change the name in some cases (I think), but it's not necessary in this case, so I guess you can delete it like clesgow suggests.